pyproject.toml

```diff
@@ -24,22 +26,25 @@ extend-safe-fixes = ["UP007", "UP006"]
 
 ignore = [
     "C90",  # mccabe
-    "FBT", # allow boolean args.
+    "FBT",  # allow boolean args.
     "CPY",  # copyright
     "S101", # allow asserts.
-    "E731",  # Allow lambda
-    "E741",  # Allow short variable names.
-    "D102",  # Do not check public method docstrings (Cannot determine overridden docstrings)
-             # @typing.override not available until python 3.12 (PEP 698)
-    "PLR",  # Too many of too many checks.
-    "ANN101",  # Ignore self type annotation.
+    "E731", # Allow lambda
+    "E741", # Allow short variable names.
+    "D102", # Do not check public method docstrings (Cannot determine overridden docstrings)
+    # @typing.override not available until python 3.12 (PEP 698)
+    "PLR",    # Too many of too many checks.
+    "ANN101", # Ignore self type annotation.
     "ANN401", # Allow Any as type annotation
-    "D105",  # Allow no docstring magic method.
+    "D105",   # Allow no docstring magic method.
     "SIM108", # Simplify if/else is not always best.
-    "PTH",  # don't require pathlib. Add eventually?
-    "TRY",  # no triceratops.
-    "NPY",  # remove numpy
-    "Q000",  # double quotes
+    "PTH123", # allow to use `open` not `Path.open`
+    "TRY",    # no triceratops.
+    "NPY",    # remove numpy
+    "Q000",   # double quotes
+    "I001",   # Ignore import order
```

Comment:
This should not be globally ignored.

Reply:
hmmm, you mean I001? or all of them
I add I001, other three are added by you and @pushfoo

Reply:
Yes i001 should not be globally set.